### PR TITLE
Fixes #618 :Developed navbar

### LINKED ISSUE #618: Problems with navbar
### Expected results
Perfect navbar
### Actual results
Problems: Eng not viewable, other bugs, lack of space
### Steps to reproduce the issue
Fix them!
## Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #618 
# Changes done in this Pull Request
Have added a drop-down to create more space, the "Eng" drop-down is back on the navbar. Linked all projects to the Projects dropdown

### CODE REVIEW

**Comment by arjunpandey2003** [comment]
deploy link

**Comment by vaibhavsingh97** [comment]
@saarthakchats Please provide us the deployment link

**Comment by saarthakchats** [comment]
https://saarthakchats.github.io/itsmethedonsi/

**Comment by saarthakchats** [comment]
@vaibhavsingh97

**Comment by arjunpandey2003** [comment]
@saarthakchats you dont have social links in your navbar

**Comment by saarthakchats** [comment]
Its in the quick link @arjunpandey2003. For space

File: _includes/header.html [hunk 1]
```diff
@@ -49,9 +49,25 @@
 		                        <li>
 		                            <a href="{{ baseurl }}./#projects">Projects</a>
 		                        </li>
-					<li>
-						<a href="{{ baseurl }} ./#fossasia-projects">Fossasia Projects</a>
-					</li>	
+								<li class="has-dropdown">
+		                            <a href="#">
```
Comment (supunTE) [inline _includes/header.html]: Indent

File: _includes/header.html [hunk 2]
```diff
@@ -49,9 +49,25 @@
 		                        <li>
 		                            <a href="{{ baseurl }}./#projects">Projects</a>
 		                        </li>
-					<li>
-						<a href="{{ baseurl }} ./#fossasia-projects">Fossasia Projects</a>
-					</li>	
+								<li class="has-dropdown">
```
Comment (supunTE) [inline _includes/header.html]: Indent

File: index.html [hunk 3]
```diff
@@ -595,7 +595,7 @@ <h3 class="mb40 mb-xs-24">FOSSASIA Google Code-In 2016/17</h3>
```
Comment (supunTE) [inline index.html:595]: remove unwanted spaces

**Review by supunTE — CHANGES_REQUESTED** [review]
Improve code quality

File: _includes/header.html [hunk 4]
```diff
@@ -83,26 +97,30 @@
...
+												</ul>
 		                                </li>
 		                                <li>
 		                                    <a href="http://events.fossasia.org/#sponsorship">
 		                                        Sponsorship
 		                                    </a>
 		                                </li>
+		                                <li>
+		                            		<a href="{{ baseurl }}./#social-links">Social Links</a>
+		                        		</li>
```
Comment (supunTE) [inline _includes/header.html:120]: no indented yet

File: _includes/header.html [hunk 5]
```diff
@@ -49,8 +49,24 @@
 		                        <li>
 		                            <a href="{{ baseurl }}./#projects">Projects</a>
 		                        </li>
-					<li>
-						<a href="{{ baseurl }} ./#fossasia-projects">Fossasia Projects</a>
+								<li class="has-dropdown">
+		                            <a href="#">
```
Comment (supunTE) [inline _includes/header.html]: indent

**Review by supunTE — CHANGES_REQUESTED** [review]
Indent those too

File: _includes/header.html [hunk 6]
```diff
@@ -49,8 +49,24 @@
...
+						<li>
+							<a href="{{ baseurl }} ./#sus">Susi.ai</a>
+						</li>
+						<li>
+							<a href="{{ baseurl }} ./#loki">Loklak Search</a>
+						</li>
+						<li>
+							<a href="{{ baseurl }} ./#other-projects">Other Sister Projects</a>
+						</li>
+						</ul>
```
Comment (supunTE) [inline _includes/header.html:69]: indent

File: _includes/header.html [hunk 7]
```diff
@@ -83,26 +97,30 @@
 		                                    </a>
```
Comment (supunTE) [inline _includes/header.html:97]: indent

**Review by supunTE — CHANGES_REQUESTED** [review]
don't makes mistakes

**Comment by saarthakchats** [comment]
Done @STEdirisuriya

**Review by supunTE — APPROVED** [review]
LGTM!

**Review by ParthS007 — APPROVED** [review]

**Review by saurabhjn76 — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**